.github/workflows/question-closer-debugger.yml

```diff
@@ -0,0 +1,29 @@
+name: Question Closer - debugger
+on:
+  schedule:
+    - cron: 20 11 * * * # Run at 11:20 AM UTC (3:20 AM PST, 4:20 AM PDT)
+  workflow_dispatch:
+   inputs:
+     readonly:
+       description: "readonly: Specify true or 1 to prevent changes from being commited to GitHub"
+       default: false
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+      - name: Install Actions
+        run: cd ./.github/actions && npm install --production && cd ../..
+      - name: Stale Closer
+        uses: ./.github/actions/StaleCloser
+        with:
+          readonly: ${{ github.event.inputs.readonly }}
+          labels: question, debugger
+          ignoreLabels: language service, internal
```

Comment:
@WardenGnaw this query also returns issues with labels like "help wanted". Would you like to ignore those?

Reply:
This query should be good enough for now. I can update it if we run into issues in the future.